sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/SwitchShouldNotBeNested.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public Program()
+        {
+            switch (choice)
+            {
+                case 'Y':
+                    Console.WriteLine("Yes");
+                    break;
+                case 'M':
+                    Console.WriteLine("Maybe");
+                    break;
+                case 'N':
+                    Console.WriteLine("No");
+                    break;
+                default:
+                    switch (value) // Noncompliant

```

Comment:
Which bit do we expect to be highlighted?

Reply:
Forgot to add the test. I'll add it (rule highlights the `switch` keyword).